test/Approximations/overapproximate_expmap.jl

```diff
@@ -51,6 +51,27 @@ for N in @tN([Float32, Float64])
         pts = sample(Pex, 10; sampler=sampler)
         @test all(p ∈ Zex for p in pts)
 
+        # overapproximate MatrixZonotopeExp
+        
+        # test degenerate case
+        A = MatrixZonotope(N[1 -2; 2 -1], [zeros(N, 2, 2)])
+        expA = MatrixZonotopeExp(A)
+        res = overapproximate(expA, MatrixZonotope, 2)
+        @test ngens(res)== 0
```

Comment:
I think in this case you should test for the exact result. Or is it approximate?